src/main/java/org/mule/extension/http/internal/HttpConnectorConstants.java

```diff
@@ -20,5 +20,6 @@
   String CONNECTOR_OVERRIDES = "Connector Overrides";
   String TLS_CONFIGURATION = "TLS Configuration";
   String AUTHENTICATION = "Authentication";
+  String STREAM_RESPONSE_PROPERTY = "mule.http.streamResponse";
```

Comment:
Is `org.mule.runtime.core.api.config.MuleProperties#SYSTEM_PROPERTY_PREFIX` visable to use?